geth/jail/jail_rpc_test.go

```diff
@@ -137,8 +146,8 @@ func (s *JailRPCTestSuite) TestSendTransaction() {
 
 	// httpRPCServer will serve as an upstream server accepting transactions.
 	httpRPCServer := httptest.NewServer(rpcService)
-	s.StartTestNode(params.RopstenNetworkID, WithUpstream(httpRPCServer.URL))
-	defer s.StopTestNode()
+	s.StartTestBackend(params.RopstenNetworkID, WithUpstream(httpRPCServer.URL))
+	defer s.StopTestBackend()
```

Comment:
This is a good remark. We already have a few tests that verify sending transactions through RPC:
1. https://github.com/status-im/status-go/blob/develop/geth/node/rpc_test.go#L68-L74
2. https://github.com/status-im/status-go/blob/develop/geth/api/backend_jail_test.go#L34

Does this test introduce anything more?

Reply:
This tests were added to validate sending transactions to the upstream RPC, do we consider the tests already present good enough and this un-necessary? @adambabik

Reply:
Oh, no. These old tests I mentioned probably do not test the upstream RPC client. But maybe we can put this test in there to keep them in the same place and be consistent?

Reply:
Sure, we can move so, should I do so in this PR? @adambabik

Reply:
We can test it here, following https://github.com/status-im/status-go/pull/276#discussion_r137028856, but I feel like it would be better to move this test to `rpc_test.go`. Then, we could have all RPC tests in one place.